pyiron_base/project/archiving/export_archive.py

```diff
@@ -52,6 +21,7 @@ def copy_files_to_archive(
         archive_directory (str): The destination directory for the archive.
         compress (bool): If True, compress the archive directory into a tarball. Default is True.
         copy_all_files (bool): If True, include all files in the archive, otherwise only .h5 files. Default is False.
+        arcname (str): The name of the archive directory. Default is the name of the directory to transfer.
```

Comment:
This one is because [`arcname` is the argument in `TarFile.add`](https://docs.python.org/3/library/tarfile.html#tarfile.TarFile.add). Otherwise yeah we could think of a better name.

Reply:
👍 consistency beats prettiness.